stratuslab/python/tar/slipstream_stratuslab/StratusLabIterClientCloud.py

```diff
@@ -103,7 +115,7 @@ def _volume_delete(self, uuid):
         wait_time = 30
         time_stop = time.time() + wait_time
         while 0 != int(pdisk.getValue('count', uuid)):
-            if time.time() <= time_stop:
+            if time.time() >= time_stop:
                 self._print_detail('Disk %s is still in use after waiting for %s sec.' %
                                    (uuid, wait_time))
```

Comment:
This is bad, but expected behaviour.  The problem is that we can't delete the disk until it gets unmounted ('count' goes to 0) after the VM is terminated.  But at the same time we can't leave disks not deleted.  Thus, I decided to continue waiting and print the message for latter debugging.  Parallel deletion should improve this.